src/domUpdates.js

```diff
@@ -0,0 +1,85 @@
+import { makeSingleChart, makeDoubleChart } from './charts.js';
+import getTodaysDate from './scripts.js';
+// ~~~~~used to render user and user data~~~~~
+const userGreeting = document.querySelector('#userGreeting');
+const userFullName = document.querySelector('#userFullName');
+const userEmail = document.querySelector('#userEmail');
+const userAddress = document.querySelector('#userAddress');
+const userStride = document.querySelector('#userStride');
+const userFriends = document.querySelector('#userFriends');
+const userStepGoal = document.querySelector('#userStepGoal');
+const averageStepGoal = document.querySelector('#averageStepGoal');
+// ~~~~~~render Sleep Method~~~~~~~~
+const lastNightSleep = document.querySelector('#lastNightSleep');
+const averageSleep = document.querySelector('#averageSleep');
+// ~~~~~charts~~~~~~~~~~
+const sleepChart = document.querySelector('#sleepChart');
+const hydrationChart = document.querySelector('#hydrationChart');
+// ~~~~~~~used to render hydration~~~~~~~~
+const dailyHydration = document.querySelector('#dailyHydration');
+// ~~~~~~~~swivel buttons~~~~~~~~~~~
+const stepCardInner = document.querySelector('#stepCardInner');
+const sleepCardInner = document.querySelector('#sleepCardInner');
+const hydrationCardInner = document.querySelector('#hydrationCardInner');
+// const swivelButton = document.querySelectorAll('.swivel-button');
+
+let domUpdates = {
```

Comment:
I like that you store all the variables inside an object and then just exported that object. It allows for better dev empathy to understand where the functions are stored from the main scripts.js file.